Add pom for E2E-Testing in Extension-Monitoring

noahrog02 opened the pull request:
No changes in functionality of the repo. This is solely for E2E testing and will not affect the repository in any way

noahrog02 committed:
Add pom 

No changes in functionality of the repo. This is solely for E2E testing and will not affect the repository in any way

stiesssh reviewed (dismissed):
works on my machine.

noahrog02 committed:
Create build.yml

This build.yml will trigger the E2E testing repository as discussed. With a "/ready-to-test" command this workflow will be activated

Things to do for Slingshot team:

Activate actions in this repository
Add a repo wide token to the secrets and name it E2E_PTA
Merge the branch with master

klinakuf committed:
modified pom to the new structure

klinakuf reviewed (approved):
this is fine now
